swf/src/avm1/read.rs

```diff
@@ -2,40 +2,32 @@
 
 use crate::avm1::{opcode::OpCode, types::*};
 use crate::error::{Error, Result};
-use crate::read::SwfReadExt;
-use crate::string::{Encoding, SwfStr, UTF_8, WINDOWS_1252};
-use byteorder::{LittleEndian, ReadBytesExt};
-use std::io::{self, Read};
+use crate::extensions::ReadSwfExt;
 
-#[allow(dead_code)]
 pub struct Reader<'a> {
     input: &'a [u8],
-    version: u8,
```

Comment:
I think it'd still be useful to keep version around, for example, in case we come across an op from future SWF versions, so that we can ignore it. Naturally this shouldn't happen in any well-formed SWF, but one actual use case is to implement a "simulate a certain Flash Player version" feature).

Reply:
Alright, I can see that! 👍